ironic.conf

```diff
@@ -5,7 +5,6 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
-enabled_bios_interfaces = idrac-wsman
```

Comment:
The correct fix would be to add `no-bios` here.

Reply:
@dtantsur should we open a PR with the configuration you are suggesting?